R/select.R

```diff
@@ -1,4 +1,5 @@
 # generalizations for the dplyr verb `select` to `scan` and `pvol` objects
+#' @exportS3Method
```

Comment:
let's hold off on exporting these, I don't think this has been thoroughly tested which is why they weren't exported just yet

Reply:
Ok, this was done to avoid getting this message when generating documentation:

```r
> devtools::document()
ℹ Updating bioRad documentation
ℹ Loading bioRad
Welcome to bioRad version 0.8.0
using vol2birdR version 1.0.3 (MistNet installed)
✖ select.R:18: S3 method `select.pvol` needs @export or @exportS3method tag.
✖ select.R:2: S3 method `select.scan` needs @export or @exportS3method tag.
```

Reply:
Bart added documentation, so I think we can expose them now.